scripts/build_schemas.sh

```diff
@@ -1,9 +1,23 @@
-#!/bin/sh 
+#!/usr/bin/env bash
+
+if [[ -x "$(command -v jsonnet)" ]]; then
+    # Build schemas for each region
+    for region_code in GB-WLS GB-ENG; do
+        SOURCE_FILE="data-source/census_individual.jsonnet"
+        # Lowercase the region code and replace '-' with '_'
+        FORMATTED_REGION_CODE=$(echo "${region_code}" | tr '[:upper:]' '[:lower:]' | tr - _)
+        DESTINATION_FILE="data-source/census_individual_${FORMATTED_REGION_CODE}.json"
+
+        jsonnet --tla-str region_code=${region_code} --tla-str census_date="2019-09-01" "${SOURCE_FILE}" > "${DESTINATION_FILE}"
```

Comment:
Currently this will not generate unique eq_id / form_type per region.

We're getting rid of eq_id / form_type, so until then quick_launch won't work.

Reply:
Have based the form type on region for now.